tokio/src/sync/notify.rs

```diff
@@ -558,6 +598,21 @@ impl Notify {
     // Alias for old name in 0.x
     #[cfg_attr(docsrs, doc(alias = "notify"))]
     pub fn notify_one(&self) {
+        self.notify_with_strategy(NotifyOneStrategy::Fifo);
+    }
+
+    /// Notifies the last waiting task.
+    ///
+    /// This function behaves identically to `notify_one` but using a
+    /// LIFO algorithm to notify waiters from the queue, if there are any.
+    ///
+    /// Check the `notify_one` documentation for more info and
+    /// examples.
+    pub fn notify_one_lifo(&self) {
```

Comment:
I'm not against calling this `notify_one_lifo`. As a consideration, do you think `notify_last_in` or `notify_one_last_in` could be better options?

Reply:
`notify_one_last_in` sounds better, users will easily understand the semantics of the method. Ill change this one.